carbon-design-system/carbon
# refactor(popover): update <div> to <span>

This PR refactors our popover to use `<span>` instead of `<div>` so that popovers may be used in inline tags like `<p>` for components like `DefinitionTooltip`. It also adds in an e2e test for VRT.

#### Changelog

**New**

- Add e2e test for Popover

**Changed**

- Change all tags to `<span>` for Popover

**Removed**

#### Testing / Reviewing

- Verify Percy has picked up the new VRT for Popover
- Verify Popover still renders as expected across browsers
